_posts/2021-02-09-open-sdg-1-2-0.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 1.2.0 released
+excerpt: Open SDG 1.2.0 is now available! Here are the details.
+author_profile: false
+---
+Open SDG, a reporting platform developed for managing and publishing data and statistics related to the UN Sustainable Development Goals (SDGs) is built with open-source libraries and tools. It is freely available for any country or organization to reuse and can be hosted and maintained using free services. Last month, the Open SDG community launched version 1.2.0, marking a significant milestone in the Open SDG journey. 1.2.0 includes a number of design upgrades and notable new features.
```

Comment:
What about this:

Open SDG, a reporting platform developed for managing and publishing data and statistics related to the UN Sustainable Development Goals (SDGs) is built with open-source libraries and tools. It is freely available for any country or organization to reuse and can be hosted and maintained using free services.

Last month, the Open SDG community launched version 1.2.0, marking a significant milestone in the Open SDG journey. Version 1.2.0 includes a number of design upgrades and notable new features.

More is in the works, with version 1.3.0 already in development along with new resources - including a series of training videos - that will make it easier for new governments to adopt Open SDG for their own needs.

Reply:
@LucyGwilliamAdmin Those edits make sense to me. I'm also in the midst of working on the video for this, and so I've now got a clear idea of the features worth mentioning. I've pushed up a rewrite of those sections, per your feedback and based on the stuff I'm putting in the video. Another round of feedback would be helpful!